src/data/mock/product/index.ts

```diff
@@ -10,20 +10,24 @@ export const productList: ProductType[] = [
     ],
     name: "SCSE Sweater - Brown",
     sizes: ["s", "m", "l", "xl"],
-    price: 25.0,
-    isAvailable: false,
+    price: 2500,
+    colorways: ["Brown"],
+    stock: [[5, 0, 3, 1, 2]],
```

Comment:
Why are there 5 entries when there are only 4 sizes? In addition, it will also make sense if this entry is merged with `SCSE Sweater - Biege`.